alacritty_terminal/src/term/mod.rs

```diff
@@ -760,6 +809,21 @@ pub struct Term<T> {
 
     /// Terminal focus
     pub is_focused: bool,
+
+    /// Last selection, stored for damage tracking
+    last_selection: Option<Selection>,
+
+    /// Last URL highlight, stored for damage tracking
+    last_highlight: Option<RangeInclusive<index::Linear>>,
+
+    /// Currently open damage rect
+    damage: DamageRect,
+
+    /// Last damage list
+    last_damage_list: Vec<DamageRect>,
```

Comment:
What's the purpose of last_damage_list? It seems like it's not really needed, am I right?

Reply:
It's a micro-optimization remnant from my initial fight for performance, saving constant allocation of the damage list. I think I'll just remove it.